1-react/store/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { createStore } from './slomux'
+
+const CHANGE_INTERVAL = 'CHANGE_INTERVAL'
+
+const reducer = (state, action) => {
+  switch(action.type) {
+    case CHANGE_INTERVAL:
+      return state += action.payload
+    default:
+      return {}
```

Comment:
Я думаю, более надёжным вариантом будет вернуть состояние в неизменном виде:

```suggestion
      return state
```

Пусть лучше в приложении ничего не произойдёт, чем оно повалится с ошибками из-за неожиданно появившегося пустого объекта.